playbooks/roles/deploy-osh/files/085-horizon.sh

```diff
@@ -23,7 +26,7 @@ set -xe
 make horizon
 
 #NOTE: Deploy command
-: ${OSH_EXTRA_HELM_ARGS:=""}
+: "${OSH_EXTRA_HELM_ARGS:=""}"
```

Comment:
shouldn't the nesting (`"<>=""}"`) be removed?

Should we really lint that upstream script?

Reply:
I guess it's good to lint it if we carry our own version. But I believe we should fix upstream instead, and not carry this file.